### PR TITLE
Add simple SELECT java based example test for Cassandra

### PR DESCRIPTION
**Pull request by maciejgrzybek**

### CODE REVIEW

File: tempto-examples/docker/docker-compose.yml [hunk 1]
```diff
@@ -30,6 +30,13 @@ services:
       POSTGRES_USER: blah
       POSTGRES_PASSWORD: blah
 
+  cassandra:
```
Comment (cawallin) [inline tempto-examples/docker/docker-compose.yml]: do you need to add cassandra to run_on_docker?

File: tempto-examples/src/main/java/com/teradata/tempto/examples/CassandraQueryTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            return immutableTable(TEST_TABLE_DEFINITION);
+        }
+    }
+
+    @Test(groups = "cassandra_query")
+    @Requires(CassandraQueryTest.ImmutableTestCassandraTable.class)
+    public void selectFromImmutableTable()
+    {
+        // Cassandra disregards the order of columns outside primary key
+        // Primary key comes first, in the order as it's defined, rest of columns are ordered alphabetically
```
Comment (cawallin) [inline tempto-examples/src/main/java/com/teradata/tempto/examples/CassandraQueryTest.java]: Maybe add another element in the primary key to show this all?

File: tempto-examples/docker/docker-compose.yml (hunk 1 above)
Comment (maciejgrzybek) [inline tempto-examples/docker/docker-compose.yml]: How differently am I suppose to run an example? Tempto-examples are executed as a part of CI.

File: tempto-examples/src/main/java/com/teradata/tempto/examples/CassandraQueryTest.java (hunk 2 above)
Comment (maciejgrzybek) [inline tempto-examples/src/main/java/com/teradata/tempto/examples/CassandraQueryTest.java]: How about 4e2a301?

**Comment by maciejgrzybek** [comment]
@cawallin: pushed 77e563f to clarify that better.
The order of input should be the same as order of columns in Cassandra, not in DDL.

File: tempto-examples/docker/docker-compose.yml (hunk 1 above)
Comment (maciejgrzybek) [inline tempto-examples/docker/docker-compose.yml]: Ouch, sorry. I didn't get your point here. No, we don't need to add it to the `run_on_docker.sh` script. It calls `${DOCKER_COMPOSE} up -d` which starts all dockers in the same compose.

**Comment by maciejgrzybek** [comment]
Merged, thanks!

**Pull request merged**